package.json

```diff
@@ -78,7 +77,6 @@
     "eslint-plugin-react": "^6.9.0",
     "express": "^4.15.2",
     "glob": "^7.1.1",
-    "husky": "^0.13.3",
```

Comment:
We were using husky for the prepublish script. We no longer need the prepublish script since we are not checking in lib.